cves/kernel/CVE-2015-6526.yml

```diff
@@ -251,8 +277,10 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
```

Comment:
This is the best way to explain in one sentence why internationalization does not apply here. I will likely steal it for my own!

Reply:
Thank you!